MangoAPI.Client/src/app/components/settings/settings.component.ts

```diff
@@ -191,9 +182,8 @@ export class SettingsComponent implements OnInit {
     }
 
     const changePassSub$ = this._usersService.changePassword(this.changePasswordCommand);
-    const response = await firstValueFrom<BaseResponse>(changePassSub$);
+    await firstValueFrom<BaseResponse>(changePassSub$);
     this.clearChangePasswordCommand();
```

Comment:
If there is an error -- then `this.clearChangePasswordCommand();` won't be called and data won't be cleared